src/NzbDrone.Common/Options/ServerOptions.cs

```diff
@@ -4,6 +4,7 @@ public class ServerOptions
 {
     public string UrlBase { get; set; }
     public string BindAddress { get; set; }
+    public string AllowedCORSOrigins { get; set; }
```

Comment:
Rename to follow our naming standard.

```suggestion
    public string AllowedCorsOrigins { get; set; }
```